contracts/FlowIDTableStaking.cdc

```diff
@@ -870,6 +870,78 @@ pub contract FlowIDTableStaking {
             }
         }
 
+        /// Set slot (count) limits for each node type (role).
+        pub fun setSlotLimits(slotLimits: {UInt8: UInt8}) {
+            FlowIDTableStaking.account.load<{UInt8: UInt8}>(from: /storage/slotLimits)
+            FlowIDTableStaking.account.save(slotLimits, to: /storage/slotLimits)
```

Comment:
```suggestion
            FlowIDTableStaking.account.save(slotLimits, to: /storage/flowStakingSlotLimits)
```
Let's try to give this a more descriptive name also

Reply:
Done.